lambdas/issuecredential/src/test/java/uk/gov/di/ipv/cri/kbv/api/service/VerifiableCredentialServiceTest.java

```diff
@@ -87,7 +87,8 @@ class VerifiableCredentialServiceTest implements TestFixtures {
         "Not Authenticated,4,1,3,0,V03",
         "Not Authenticated,2,0,2,0,V03",
         "Unable to Authenticate,3,2,1,0,V03",
-        "Unable to Authenticate,2,0,2,0,",
+        "Unable to Authenticate,2,0,2,0,V03",
+        "Unable to Authenticate,2,1,1,0,V03",
```

Comment:
This should now be failing, if you looked at the slack thread, and made the changes I recommended, it is good to have them in initially and ensure they are both failing. 

```
       "Unable to Authenticate,2,0,2,0,",
        "Not Authenticated,2,0,2,0,",
```
Both of those should pass, but really they should actually not be there at all `b'cos no questionSummary` comes back from < 3 questions so we should remove them

Reply:
I did see the thread and I agree that we should just remove them - not really an accurate representation of what comes through